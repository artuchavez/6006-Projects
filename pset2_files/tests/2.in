max-heap
Abby,Betty,Cathy,Demi,Elly,Fermi
5,0.7,4,0,4.3,2.1
Cathy
1.1
