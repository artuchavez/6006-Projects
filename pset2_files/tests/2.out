Abby,Betty,Cathy,Demi,Elly,Fermi
5,0.7,1.1,0,4.3,2.1